src/components/Home.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { createStore, applyMiddleware } from 'redux';
+import { Provider } from 'react-redux';
+// import { store } from '../redux/configureStore';
```

Comment:
### [OPTIONAL]

- [ ] It will be great to delete commented code. This will keep your code clean. Remember, you can always use git history to go back to the previous version of your code.